dashboard/dashapi/dashapi.go

```diff
@@ -400,6 +401,11 @@ type BugReport struct {
 	BisectCause    *BisectResult
 	BisectFix      *BisectResult
 	Assets         []Asset
+	Subsystems     []*BugSubsystem
```

Comment:
Earlier you used Subsystems []BugSubsystem.
Here it is Subsystems     []*BugSubsystem.
I like pointer-less one more.